extension/trace.h

```diff
@@ -84,16 +84,13 @@ static zend_always_inline zend_string *hp_get_trace_callback(zend_string *functi
         callback = (hp_trace_callback*)zend_hash_find_ptr(XHPROF_G(trace_callbacks), function_name);
         if (callback) {
             trace_name = (*callback)(function_name, data);
-        } else {
```

Comment:
Previously:

- If a callback is found, release function_name and return trace_name
- If no callback is found, return function_name directly without releasing it